src/pages/project/tests/projectContributorHelpers.test.ts

```diff
@@ -0,0 +1,210 @@
+import { describe, expect, it } from 'vitest';
+import { addContributor, AddContributorErrors } from '../helpers/projectContributorHelpers.js';
+import {
+  abcOrgAsAffiliation,
+  contributorsArrayWithProjectManager,
+  selectedPersonIdentity,
+  selectedPersonWithAffiliation,
+  selectedPersonWithoutAffiliation,
+} from './mockObjects';
+
+describe('addContributor', () => {
+  describe('when adding a project manager with affiliation', () => {
+    it.skip('if there is already a project manager in the contributors array it returns an error', async () => {
```

Comment:
bevisst at denne skippes?
```suggestion
    it('if there is already a project manager in the contributors array it returns an error', async () => {
```

Reply:
Ja, for det er ikke implementert i addContributor enda. Egentlig vil dette aldri skje fordi buttonen er disablet. Men greit å håndtere det hvis det skjer uansett. Men siden det ikke er en bug så fikser jeg det senere.